### ISSUE TITLE
Evaluating without Human3.6M dataset?

### ISSUE THREAD

**Issue by BartArtanim** [karfly/learnable-triangulation-pytorch#71]
Dear all, I am currently trying to evaluate your excellent work and ultimately hope to test it on some of my own set of data. 

The main issue I'm facing at the moment is that I can't seem to get access to the Human3.6M dataset. For obvious reasons you can't supply this, but I have found that you do supply some of the training results for download on Google Drive. I do however have a hard time piecing together if I can run tests on other data without the Human3.6M dataset.

For example, I get an indication that `human36m-multiview-labels-GTbboxes.npy` is required, which is a result of step 6 [in the pre-process here](https://github.com/karfly/learnable-triangulation-pytorch/blob/master/mvn/datasets/human36m_preprocessing/README.md). And that I can't get because I don't have the processed data. 

I have read in previous issues that retraining is not necessary for other datasets, so at least I could perhaps test it on the CMU dataset (as you have done) or ultimately go for my own data. But I would appreciate it if you could let me know if I have run upon a dead end, or if there are steps I can follow without having the Human3.6M dataset at all.

**Comment by shrubb** [comment]
I'm not sure, but feel like it makes more sense to transfer a CMU model to Human3.6M rather than the other way around. @karfly what do you think?

I believe you should better look for some more multi-view datasets where full body is visible (maybe sports, football?...). You have good chances of running our pre-trained model there.

**Comment by antithing** [comment]
@BartArtanim Hi! Did you find a suitable data set to run this code on?

**Comment by BartArtanim** [comment]
Hi @antithing. Unfortunately not. And I no longer have the time to really look into this. I have abandoned the evaluation for now. If I get back to it as some point I'll report back.

**Comment by yangtao19920109** [comment]
hi! I  do think you have a excellent work.so I want to  run your model on by myself.but unfortunately,I have not the human3.6 datasets. so can you give a inference.py file.then I can get the data formate and I can easily run the other datasets
